cp3pt0-deployment/common/migrate_singleton.sh

```diff
@@ -53,18 +55,11 @@ function main() {
     delete_operator "ibm-cert-manager-operator" "$OPERATOR_NS"
     
     if [[ $ENABLE_LICENSING -eq 1 ]]; then
-
-        is_exists=$("$OC" get deployments ibm-licensing-operator -n "$OPERATOR_NS")
```

Comment:
Looks like the code to migrate License Service is now removed, so are we regressing here the License Service upgrade, by adding Reported migrate? Pls clarify - thx.

Reply:
restored